_source/logzio_collections/_prometheus-sources/java-custom-metrics-micrometer.md

```diff
@@ -0,0 +1,200 @@
+---
+title: Send custom metrics from your Java code with Micrometer
+logo:
+  logofile: micrometer.svg
+  orientation: vertical
+open-source:
+  - title: Micrometer custom metrics
+    github-repo: micrometer-registry-logzio
+    maven: test
+data-source: Micrometer for custom metrics
+flags:
+  logzio-plan:  
+  beta: false
+templates: ["docker"]
+contributors:
+  - yotamloe
+  - yberlinger
+shipping-tags:  
+  - prometheus
+---
+
+To send custom metrics from your Java application to Logz.io, you'll need to use a [Micrometer metrics](https://micrometer.io/) registry for dimensional metrics.
+
+This registry can be used by any application that uses Micrometer for recording metrics.
+
+## Usage:
```

Comment:
Does this section need further explanations? 
Why has the heading skipped from H2 to H4 ?

Reply:
Maybe change the following sentences:
To send custom metrics from your Java application to Logz.io, you’ll need to use a Micrometer metrics registry for dimensional metrics.

This registry can be used by any application that uses Micrometer for recording metrics

To :
Micrometer (add link to micrometer site) provides a simple facade over the instrumentation clients for the most popular monitoring systems, allowing you to instrument your JVM-based application code without vendor lock-in. 
Logz.io's Micrometer metrics registry sends your custom metrics to your Logz.io account.

Reply:
@Doron-Bargo -
Is the suggested change relevant for both topics or only for `_source/logzio_collections/_prometheus-sources/java-custom-metrics-micrometer.md`?